.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     - name: Install Dependencies
       run: |
-        python -m pip install -U jupyterlab~=3.0 jupyter_packaging~=0.10
+        python -m pip install -U jupyterlab~=3.0 jupyter_packaging~=0.10 papermill
```

Comment:
Looks like you may need to add this here: https://github.com/voila-dashboards/voila/blob/main/setup.cfg#L49-L56 instead

Reply:
I believe this is where the test deps get installed for CI: https://github.com/voila-dashboards/voila/blob/de712eae5e64ed880177cc80e21f9c77e4b1a02e/.github/workflows/main.yml#L42

Link to CI error line for easy access: https://github.com/voila-dashboards/voila/runs/6181097089?check_suite_focus=true#step:6:93

Reply:
Ah and on the windows tests it uses this line instead: https://github.com/voila-dashboards/voila/blob/de712eae5e64ed880177cc80e21f9c77e4b1a02e/.github/workflows/main.yml#L85